packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -875,7 +875,42 @@ const schema: SchemaType<"Comments"> = {
     canRead: ['guests'],
     canUpdate: ['sunshineRegiment', 'admins'],
     canCreate: ['members', 'sunshineRegiment', 'admins'],
-  } 
+  },
+
+  doppelComments: {
+    // Resolved by doppelComments resolver
+    type: Array,
+    optional: true,
+    canRead: ['guests'],
+    canCreate: ['admins'],
+    canUpdate: ['admins'],
+    hidden: true,
+  },
+
+  'doppelComments.$': {
+    type: String,
```

Comment:
I think this is wrong, since the resolver in `commentResolvers.ts` returns an array of DoppelComments.  Should probably be `type: Object`.